src/Listener.js

```diff
@@ -0,0 +1,63 @@
+const desktopIdle = require('desktop-idle');
+
+// We want the status of the user to be accurate.
+// To ensure this we need to frequently check the users status
+const DEFAULT_POLLING_TIMEOUT = 1000;
+
+class Listener {
+  constructor(id, timeToAway, callback) {
```

Comment:
Hm, Im not keen on using the word `time` because it is a little ambiguous. The value is not a time value, it is a number. `secondsUntilAway` might be better? 

Or maybe `inactiveDuration` (how long (`duration`) should we wait until we consider a user inactive)

Reply:
`secondsUntilAway` would be my preference 👍